src/utils/docker.rs

```diff
@@ -0,0 +1,55 @@
+use bollard::container::{Config, CreateContainerOptions};
+use bollard::Docker;
+
+use bollard::image::CreateImageOptions;
+use futures_util::TryStreamExt;
+
+pub fn run_docker_image(image: &str, container_name: &str, env_vars: Vec<&str>) {
+    is_docker_installed();
+    match pull_and_start_docker_image(image, container_name, env_vars) {
+        Ok(..) => {
+            log::info!("Successfully ran {}", container_name);
+        }
+        Err(err) => {
+            log::error!("Error: {} running image: {}", err, image);
+        }
+    };
+}
+
+#[tokio::main]
+async fn is_docker_installed() -> bool {
+    let docker = Docker::connect_with_local_defaults().unwrap();
+    return match docker.version().await {
+        Ok(_) => {
+            log::info!("Docker running!");
+            true
+        }
+        Err(_) => {
+            panic!("Please check docker installation, panicking");
+        }
+    };
+}
+
+#[tokio::main]
+async fn pull_and_start_docker_image(
+    image: &str,
+    container_name: &str,
+    env_vars: Vec<&str>,
+) -> Result<(), Box<dyn std::error::Error + 'static>> {
+    let docker = Docker::connect_with_local_defaults().unwrap();
+
+    docker
```

Comment:
this is creating a new image, i think we should directly give the name of the image in the config and create a container.

Reply:
It fails with following error in case the image is not present locally already so I think it is required.

```
Error: Docker responded with status code 404: No such image: ghcr.io/lambdaclass/stark_compass_explorer:v0.2.34.2 running image: ghcr.io/lambdaclass/stark_compass_explorer:v0.2.34.2
```